mockgcp/mocknetworksecurity/authorizationpolicy.go

```diff
@@ -119,10 +119,10 @@ func (s *NetworkSecurityServer) UpdateAuthorizationPolicy(ctx context.Context, r
 		RequestedCancellation: false,
 		Target:                name.String(),
 		Verb:                  "update",
-		ApiVersion:            "v1",
+		ApiVersion:            "v1beta1",
```

Comment:
Ah, that's where it comes from.  Why?  Should we add a gcloud test?  Or maybe split out the mockgcp changes?